examples/cockroachdb/README.md

```diff
@@ -0,0 +1,263 @@
+# CockroachDB
+
+[CockroachDB](https://www.cockroachlabs.com/) is one of the database servers based on SQL.
+
+## Introduction
+
+This chart bootstraps a three node Cockroachdb deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.16+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.80.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
+## Installing the Chart
+
+To install the CockroachDB database using the `cockroachdb` chart with the release name `cockroachdb-release`:
+
+```bash
+# Add cockroachdb in your local chart repository
+$ helm repo add cockroachdb https://charts.cockroachdb.com/
+
+# Update your local chart repository
+$ helm repo update
+
+# Install the cockroachdb in cockroachdb namespace
+$ helm install cockroachdb-release --namespace cockroachdb --create-namespace cockroachdb/cockroachdb
+
+# Install the secure cockroachdb client
+$ curl -OOOOOOOOO https://raw.githubusercontent.com/cockroachdb/helm-charts/master/examples/client-secure.yaml
+
+# Edit Fields in client-secure.yaml
+spec.serviceAccountName: cockroachdb-release
+spec.image: cockroachdb/cockroach:<VERSION>
```

Comment:
What are the correct possible versions one can use?
Can we provide a sample yaml manifest?